Description: .loadfontloop must be an operator
 In the fix for Bug 699816,
 I omitted to make .loadfontloop into an operator,
 to better hide .forceundef and .putgstringcopy.
Origin: upstream, http://git.ghostscript.com/?p=ghostpdl.git;h=a5a9bf8
Author: Chris Liddell <chris.liddell@artifex.com>
Forwarded: yes
Bug: http://bugs.ghostscript.com/show_bug.cgi?id=699938
Bug-Debian: https://bugs.debian.org/911175
Bug-CVE: https://security-tracker.debian.org/tracker/CVE-2018-18284
Last-Update: 2018-10-18
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Resource/Init/gs_fonts.ps
+++ b/Resource/Init/gs_fonts.ps
@@ -1148,7 +1148,7 @@
 
     } loop              % end of loop
 
- } bind executeonly def % must be bound and hidden for .putgstringcopy
+ } bind executeonly odef % must be bound and hidden for .putgstringcopy
 
 currentdict /.putgstringcopy .undef
 
